studio/components/interfaces/Home/ProjectList/ProjectList.tsx

```diff
@@ -32,9 +35,24 @@ const ProjectList: FC<Props> = ({ rewriteHref }) => {
         const isEmpty = sortedProjects?.length == 0
         const canReadProjects = checkPermissions(PermissionAction.READ, 'projects', undefined, id)
 
+        const overdueInvoices = allOverdueInvoices?.filter((it) => it.organization_id === id) || []
```

Comment:
perhaps `(allOverdueInvoices || [])`
otherwise `overdueInvoices` can be undefined (while loading) and L45 will error out

Reply:
Adjusted it